test/localhub/lock_test.go

```diff
@@ -0,0 +1,182 @@
+package localhub
```

Comment:
I saw that you converted `when()`, `then()`, `scenario()` from gingko to Go comment. Consider using `t.Log()`, at least for the scenario headers - sometimes it can improve debugging and following the test progress 🙂